providers/ndb.rb

```diff
@@ -35,7 +35,6 @@
     code <<-EOF
     set -e
     cd #{flyway_dir}
-    #{node['ndb']['scripts_dir']}/mysql-client.sh #{node['hops']['db']} < #{node['hops']['dir']}/ndb-hops/flyway.sql
```

Comment:
I think this line was here to cover for another issue. Yes, flyway creates the table by default, that's what we do in Hopsworks, but it creates it on `InnoDB`, which means it creates issues for backups/upgrades.

Can you figure out how to get flyway to create the schema table on ndb before merging this PR?

Reply:
Is it related to  https://logicalclocks.atlassian.net/browse/HOPSWORKS-1867 ?